src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java

```diff
@@ -27,39 +27,39 @@
   private static final String ERROR_MESSAGE =
       "We don't recognize these credentials. Try again or register a new account.";
 
-  @Test(groups = "MercuryLoginTest_002")
+  @Test(groups = {"MercuryLoginTest_002", "LoginTests_001"})
   public void MercuryLoginTest_002_userCanNotLogInWithWrongPassword() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)